src/components/inject.js

```diff
@@ -57,28 +57,28 @@ Please be sure the component that calls createSource or createToken is within an
     } else {
       throw new Error(
         `You did not specify the type of Source or Token to create.
-        We could not infer which Element you want to use for this operation.`
+        We could not infer which Element you want to use for this operation.`,
       );
     }
-  }
+  };
   wrappedCreateToken = (options: mixed = {}) => {
     if (options && typeof options === 'object') {
-      const {type, ...rest} = options;
-      const element = this.findElement(type);
+      const {elementType, ...rest} = options;
```

Comment:
As far as I can se the type was never defined either? 
It is just used by the findElement.

Related:
What is the purpose of the variable type here: https://github.com/stripe/react-stripe-elements/blob/master/src/components/Element.js#L17

Could that variable be renamed as to prevent confusion of what `type` is? or could we add that the createSource takes something like:
```json
{
  type: 'card',
  sourceParams: {
    type: 'three_d_secure'
  }
}
```